package.json

```diff
@@ -16,39 +17,39 @@
 	"author": "@kianenigma",
 	"license": "ISC",
 	"devDependencies": {
-		"@babel/cli": "^7.23.0",
-		"@babel/core": "^7.23.2",
-		"@babel/preset-typescript": "^7.23.2",
-		"@types/jest": "^29.5.7",
-		"@types/js-yaml": "^4.0.8",
-		"@types/node": "^20.8.10",
-		"@types/nodemailer": "6.4.13",
-		"@types/request": "^2.48.11",
-		"@types/yargs": "^17.0.29",
-		"@typescript-eslint/eslint-plugin": "^6.9.1",
-		"@typescript-eslint/parser": "^6.9.1",
-		"eslint": "^8.52.0",
-		"eslint-config-prettier": "^9.0.0",
-		"eslint-plugin-prettier": "^5.0.1",
+		"@babel/cli": "^7.23.9",
+		"@babel/core": "^7.23.9",
+		"@babel/preset-typescript": "^7.23.3",
+		"@types/jest": "^29.5.11",
+		"@types/js-yaml": "^4.0.9",
+		"@types/node": "^20.11.15",
+		"@types/nodemailer": "6.4.14",
+		"@types/request": "^2.48.12",
+		"@types/yargs": "^17.0.32",
+		"@typescript-eslint/eslint-plugin": "^6.20.0",
+		"@typescript-eslint/parser": "^6.20.0",
+		"eslint": "^8.56.0",
+		"eslint-config-prettier": "^9.1.0",
+		"eslint-plugin-prettier": "^5.1.3",
 		"jest": "^29.7.0",
-		"nodemon": "^3.0.1",
-		"npm-check-updates": "16.14.6",
-		"pm2": "^5.3.0",
-		"prettier": "^3.0.3",
+		"nodemon": "^3.0.3",
+		"npm-check-updates": "16.14.14",
+		"pm2": "^5.3.1",
+		"prettier": "^3.2.4",
 		"ts-interface-builder": "^0.3.3",
-		"ts-jest": "^29.1.1",
-		"ts-loader": "^9.5.0",
-		"ts-node": "^10.9.1",
-		"typescript": "5.2.2"
+		"ts-jest": "^29.1.2",
+		"ts-loader": "^9.5.1",
+		"ts-node": "^10.9.2",
+		"typescript": "5.3.3"
 	},
 	"dependencies": {
-		"@polkadot/api": "10.10.1",
+		"@polkadot/api": "10.11.2",
 		"js-yaml": "4.1.0",
-		"matrix-js-sdk": "^29.1.0",
-		"nodemailer": "^6.9.7",
+		"matrix-js-sdk": "^31.2.0",
```

Comment:
this sounds like a breaking change though, just judging by its major version bump.